tests/unit/CMakeLists.txt

```diff
@@ -46,6 +46,13 @@ target_link_libraries(sentry_test_unit PRIVATE
 	"$<$<OR:$<PLATFORM_ID:Linux>,$<PLATFORM_ID:Android>>:-Wl,-E>"
 )
 
+# This is a bit unfortunate: Building the unit tests also builds and links the
+# backend, which is unnecessary for the unit tests themselves,
+# and takes a long time to compile.
```

Comment:
If we don't unit-test the crashpad backend, can we just remove that test build entirely?

Reply:
not with additional logic :-(
the unit tests "inherit" all the sources from `sentry`.

When using pytest, we compile the unit tests correctly anyway. But you were mentioning errors that had `constexpr` in them, and I was able to reproduce them by just building normally without any extra flags, which builds crashpad, and the tests. So better to make it work with the default cmake flags.